NanoProd/python/customize.py

```diff
@@ -15,7 +15,7 @@ def customize(process):
 
 def customize_pnet(process):
     process = customize(process)
-
+    process.finalTaus.cut = 'pt > 0'
```

Comment:
I have `process.finalTaus.cut = cms.string("tauID('decayModeFindingNewDMs')")`. The decayModeFinding doesn't exist as a branch in latest NanoAOD because it's required by default. This shouldn't have an impact on PNet results, right?

Reply:
I think this cut is already applied at the miniAOD level. But I can put it on top, just for safety